invoke.go

```diff
@@ -11,10 +12,13 @@ import (
 // A user defined call stub.
 type APICall func(context.Context) error
 
-// scaleDuration returns the product of a and mult.
-func scaleDuration(a time.Duration, mult float64) time.Duration {
-	ns := float64(a) * mult
-	return time.Duration(ns)
+// scaleDuration returns the duration in `mult` after `d`
```

Comment:
This seems like an unrelated change.
Also, don't use markdown in the comments.

// scaleDuration scales the duration d by mult.